app/src/main/java/com/konkuk/arabyte_aos/presentation/ui/home/HomeScreen.kt

```diff
@@ -1,8 +1,342 @@
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.LocalLifecycleOwner
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.flowWithLifecycle
+import com.konkuk.arabyte_aos.R
+import com.konkuk.arabyte_aos.presentation.type.component.ArabyteCategoryType
+import com.konkuk.arabyte_aos.presentation.ui.component.ArabyteNoticeBoardItem
+import com.konkuk.arabyte_aos.presentation.ui.component.ArabyteReviewItem
+import com.konkuk.arabyte_aos.presentation.ui.component.button.ArabyteCategoryButton
+import com.konkuk.arabyte_aos.presentation.ui.home.component.HomeContentTitle
+import com.konkuk.arabyte_aos.presentation.ui.home.component.HomePagerIndicator
+import com.konkuk.arabyte_aos.presentation.ui.home.component.HomeRegionChip
+import com.konkuk.arabyte_aos.ui.theme.ArabyteAOSTheme
+import com.konkuk.arabyte_aos.ui.theme.ArabyteTheme
 
 @Composable
-fun HomeScreen(modifier: Modifier = Modifier) {
+fun HomeRoute(
+    onNavigateToNoticeBoard: () -> Unit,
+    onNavigateToNoticeBoardDetail: (Int) -> Unit,
+    onNavigateToReview: () -> Unit,
+    onNavigateToReviewDetail: (Int) -> Unit,
+    onNavigateToReviewCategory: (ArabyteCategoryType) -> Unit,
+    onNavigateToMyPage: () -> Unit,
+    modifier: Modifier = Modifier,
+    viewModel: HomeViewModel = hiltViewModel(),
+    innerPaddingValues: PaddingValues = PaddingValues(0.dp),
+) {
+    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
+    val lifecycleOwner = LocalLifecycleOwner.current
+
+    LaunchedEffect(Unit) {
+        viewModel.setEvent(HomeContract.HomeEvent.LoadUserName)
+    }
+    LaunchedEffect(Unit) {
+        viewModel.setEvent(HomeContract.HomeEvent.LoadRegion)
+    }
+    LaunchedEffect(Unit) {
+        viewModel.setEvent(HomeContract.HomeEvent.LoadReviewList)
+    }
+    LaunchedEffect(Unit) {
+        viewModel.setEvent(HomeContract.HomeEvent.LoadNoticeBoardList)
+    }
+    LaunchedEffect(viewModel.sideEffect, lifecycleOwner) {
+        viewModel.sideEffect.flowWithLifecycle(lifecycle = lifecycleOwner.lifecycle)
+            .collect { sideEffect ->
+                when (sideEffect) {
+                    is HomeContract.HomeSideEffect.NavigateToNoticeBoard -> onNavigateToNoticeBoard()
+                    is HomeContract.HomeSideEffect.NavigateToNoticeBoardDetail -> onNavigateToNoticeBoardDetail(sideEffect.noticeBoardId)
+                    is HomeContract.HomeSideEffect.NavigateToReview -> onNavigateToReview()
+                    is HomeContract.HomeSideEffect.NavigateToReviewCategory -> onNavigateToReviewCategory(sideEffect.categoryType)
+                    is HomeContract.HomeSideEffect.NavigateToReviewDetail -> onNavigateToReviewDetail(sideEffect.reviewId)
+                    is HomeContract.HomeSideEffect.NavigateToMyPage -> onNavigateToMyPage()
+                }
+            }
+    }
+
+    HomeScreen(
+        onNavigateToNoticeBoard = onNavigateToNoticeBoard,
```

Comment:
아 다시보니 이부분을 여쭤보신 것 같네요 이렇게 함수를 바로 screen에 넘겨준다면 SideEffect로 관리하는 의미가 없을것같아요... 
예시로 하나만 수정해 드리자면
```suggestion
    HomeScreen(
        onNavigateToNoticeBoard = {viewModel.setSideEffect(HomeContrat.HomeSideEffect.NavigateToNoticeBoard)},
```
이런식으로 HomeScreen에서 onNavigateToNoticeBoard 함수가 호출되었을 때 SideEffect가 변경되고 Route의 LaunchedEffect에서 변경을 감지하여 when문을 통해 onNavigateToNoticeBoard( ) 가 실행될거에요!